mythril_core/src/vm.rs

```diff
@@ -300,9 +300,9 @@ mod test {
                 "read_file not implemented for TestVmServices".into(),
             ))
         }
-        fn acpi_addr(&self) -> Result<HostPhysAddr> {
+        fn rsdp(&self) -> Result<crate::acpi::rsdp::RSDP> {
```

Comment:
Minor change: can you `use acpi` and then have this as just `acpi:rsdp::RSDP` instead of using the full `crate` path here (and below as well).

Reply:
Looks like I had to use `use crate::acpi` inside of vm.rs